CHANGELOG.md

```diff
@@ -14,9 +14,6 @@
 * Advanced search on emptiness for CVE ID (`OSIDB-2806`)
 * Embedded Alerts on top of the Flaw form (`OSIDB-1211`)
 
-### Changed
-* Use comment_zero instead of description from OSIDB (`OSIDB-2784`)
```

Comment:
Why are we deleting the chagelog from OSIDB-2784 and not adding a new one for OSIDB-2785?

Reply:
It was a reaction on the Carlos previous point here https://github.com/RedHatProductSecurity/osim/pull/242#discussion_r1625552686 

This is not a user facing thing, for user nothing changes, as it is already a `Description` field, we just change the field which is the source for the data and the same case was for the OSIDB-2784, user already sees `Comment #0` in OSIM, we just changed the field source, so there should have been no changelog entries, so just retrospectively deleting this one.